### DISCUSSION TITLE
CrudService.getRepository getter 관련

### DISCUSSION THREAD

**Discussion by woowahan-kibae** [woowabros/nestjs-library-crud#24] [Ideas]
CrudService에서 getRepository 라는 public getter로 저장소를 노출하고 있습니다.
get accessor 이름으로 getRepository 는 안맞는것 같아서 아래 두가지 방식 중 하나를 제안해 봅니다~

- getter가 아닌 getRepository() 메쏘드로 바꾸기
```ts
getRepository(): Repository<T> {
    return this.repository;
}
```

- getRepository가 아닌 CrudService.repository로 바꾸기(readonly 만 추가)
```ts
constructor(protected readonly repository: Repository<T>) {...}
```

**Comment by jiho-kr** [comment]
앗, 이런! 🙀 
`breaking change`가 발생하긴 하겠지만 빠르게 2번 안으로 수정하는것에 한 표를 놓아봅니다

**Reply by woowahan-kibae** [reply]
저도 2번 추천요~ ㅎㅎ
repository를 어디서 가져와야 하거나 조립해야 한다면 1번이지만 생성자에서 주입받는것을 주는거라서 2번 추천요~

**Comment by woowahan-kibae** [comment]
https://github.com/woowabros/nestjs-library-crud/pull/27
웹스톰의 힘으로 간단히 수정해 보았습니다 ㅎㅎ
테스트 케이스는 다 통과하네요~
